lib/src/scss-shared/_nx-form-validation.scss

```diff
@@ -5,20 +5,53 @@
  * distribution and is available at https://www.eclipse.org/legal/epl-2.0/.
  */
 @import 'nx-variables';
+@import 'nx-container-helpers';
 
 @mixin valid() {
-  border-left-color: $nx-valid-color;
-  border-left-style: solid;
+  .nx-text-input__box {
+    border-color: $nx-valid-color;
+  }
+
+  .nx-icon--valid {
+    display: inline-block;
+  }
 }
 
 @mixin invalid() {
-  border-color: $nx-invalid-color;
-  border-left-style: solid;
+  .nx-text-input__box {
+    border-color: $nx-invalid-color;
+  }
+
+  .nx-icon--invalid {
+    display: inline-block;
+  }
 }
 
 @mixin pristine() {
-  border-color: $nx-grey-4;
-  border-left-color: $nx-valid-color;
-  border-left-style: double;
-  color: $nx-text-color-light;
+  .nx-text-input__box {
+    border-color: #d1d6f0;
+    color: $nx-text-color-light;
+  }
+
+  .nx-icon--invalid, .nx-icon--valid {
+    display: none;
+  }
+}
+
+.nx-icon--invalid, .nx-icon--valid {
```

Comment:
Everything from here down in this file is emitting, so cannot be in this file since this is in the `scss-shared` dir.

Reply:
> Everything from here down in this file is emitting, so cannot be in this file since this is in the `scss-shared` dir.

Fixed here: https://github.com/sonatype/sonatype-react-shared-components/pull/57/commits/d734699347f1f2d14bfce7f33d24ff38a7d6d3ea

Reply:
> Fixed here: d734699

Missed the one this comment is on

Reply:
> > Fixed here: [d734699](https://github.com/sonatype/sonatype-react-shared-components/commit/d734699347f1f2d14bfce7f33d24ff38a7d6d3ea)
> 
> Missed the one this comment is on

https://github.com/sonatype/sonatype-react-shared-components/pull/57/commits/351e88cf94fb709cad84ec6f4724eeb5694ee72a